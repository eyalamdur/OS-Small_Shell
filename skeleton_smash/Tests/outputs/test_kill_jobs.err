smash error: kill: invalid arguments
smash error: kill: invalid arguments
terminate called after throwing an instance of 'std::invalid_argument'
  what():  stoi
