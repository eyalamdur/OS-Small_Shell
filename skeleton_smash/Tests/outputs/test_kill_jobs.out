smash> submitted by Eran
smash> smash> smash> smash> [1] ./printSignals.exe &
[2] sleep 200&
[3] sleep 300&
smash> signal number 9 was sent to pid 12696
smash> smash> [1] ./printSignals.exe &
[3] sleep 300&
[4] sleep 400&
smash> smash> smash> smash> smash> smash> signal number 9 was sent to pid 12861
smash> signal number 5 was sent to pid 12695
smash> received signal number: 5
smash> [1] ./printSignals.exe &
[3] sleep 300&
[4] sleep 500&
smash> smash: sending SIGKILL signal to 3 jobs:
12695: ./printSignals.exe &
12697: sleep 300&
13012: sleep 500&
